File: ans2-5.js
function twoSum(nums, target){
    // your code here
    for(let i = 0; i <= nums.length; i++){
        if(target-nums[i] in nums){
            data = [i,nums.indexOf(target-nums[i])];
            return data;
        }
    }
    console.log(data);
    
}
let result=twoSum([2, 11, 7, 15], 9);
console.log(result); // show [0, 2] because nums[0]+nums[2] is 9

File: ans2-5.py
# -*- coding: utf-8 -*-
"""
Created on Mon Sep 26 17:01:58 2022

@author: 劉佳怡
"""

def twoSum(nums, target):
    # your code here
    for i in nums :
        ans = 0
        ans = target - i
        if(ans in nums):
            ansIndex1 = nums.index(ans)
            ansIndex2 = nums.index(i)
            return[ansIndex2,ansIndex1]
        
result=twoSum([2, 11, 7, 15], 9)
print(result) # show [0, 2] because nums[0]+nums[2] is 9
